endpoint/BUILD

```diff
@@ -4,7 +4,7 @@ cc_library(
     hdrs = ["endpoint_core.h"],
     defines = [
         "ENDPOINT_HOST=\"localhost\"",
-        "ENDPOINT_PORT=\"443\"",
+        "ENDPOINT_PORT=\"8000\"",
```

Comment:
Minimize the functional changes. If you really have to change the port number, there must be an informative description along with considerations.